src/helpers/helpers-test.ts

```diff
@@ -0,0 +1,27 @@
+import { isArgFormatValid } from './helpers';
+
+const shortArg = '-t';
+const longArg = '--test';
+const extraLongArg = '--test-run';
+const invalidShortArg = '--t';
+const invalidLongArg = '-test';
```

Comment:
One more thing: if there's no string reuse, don't bother making `const` of these things, there's no benefit in extracting a const , and

```ts
expect(isArgFormatValid('-t')).toEqual(true);
```

... is actually more readable (since there's one less mental indirection to do) than

```ts
const shortArg = '-t';
expect(isArgFormatValid(shortArg)).toEqual(true);
```

Reply:
I thought about doing that but followed the example in the current `app/src/helpers/helpers.test.ts`
I'll make the change!

Reply:
> I thought about doing that but followed the example in the current `app/src/helpers/helpers.test.ts`. I'll make the change!

👍! Yeah, but see: in the `linkIsInternal`, there is actual cross-test reuse of for example the `internalUrl` constant. So in this case, it makes total sense to extract it into a variable, because you don't want to duplicate it.